zastosowania-komputerow-w-elektrotechnice.csl

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
```

Comment:
So, given that you have hardcoded some terms into English into the style, I suggest we set the style to English as a default: add ```default-locale="en-US"``` to this line (line 2).